Added "utils" directory for ethond system utils with `reg16`

VGeo opened the pull request:
(utility for reading and writing SoC FPGA registers).

Commit:
Added "utils" directory for ethond system utils with `reg16` (utility for
    reading and writing SoC FPGA registers).

yapavelkurochkin reviewed (commented):
в целом всё ок, но есть пожелания (1е - самое важное):
- формат чисел (адресов и данных) не описан нигде, ни в help, ни в README, нигде.
- а man будет собран и поставлен пакетом? что-то не похоже ;)
- коды возврата странные какие-то... зачем мы их экранируем? вместо нормальных возвращаем -1, -2... это не гуд
- лучше make: нет смысла для программы в один сишник юзать automake

Володь, а без форка никак? у тебя не прав сделать бренч в этой репке?

VGeo commented:
Все поправим.
Я вчера куда только не тыкал и получал You have no permissions. Более
бесправный, видимо, только тот, кто на гитхабе вообще не зарегистрирован.

5 октября 2016 г., 22:14 пользователь Pavel Kurochkin <
notifications@github.com> написал:

> *@yapavelkurochkin* commented on this pull request.
>
> в целом всё ок, но есть пожелания (1е - самое важное):
>
>    - формат чисел (адресов и данных) не описан нигде, ни в help, ни в
>    README, нигде.
>    - а man будет собран и поставлен пакетом? что-то не похоже ;)
>    - коды возврата странные какие-то... зачем мы их экранируем? вместо
>    нормальных возвращаем -1, -2... это не гуд
>    - лучше make: нет смысла для программы в один сишник юзать automake
>
> Володь, а без форка никак? у тебя не прав сделать бренч в этой репке?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/STC-Metrotek/ethond/pull/1#pullrequestreview-2983497>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AKO4pn0h9G_m_JPGeXRTe6qy-qglSfu1ks5qw_cqgaJpZM4KO5rx>
> .
>

Commit:
1. Use pure make instead of autotools
2. Removed dbg version of package
3. Added reg16 parameters description and usage examples to README
4. Use correct error codes in reg16
5. Added manpage

yapavelkurochkin reviewed (approved):
Ура, пакетик собирается. Программа возвращает коды ошибок. 

Правда, dpkg-buildpackage падает из-за отсутствия подписи и если прав доступа к /dev/mem нет, то пишет "open: permission denied"

Как думаете, догадается юзер подписать и вызвать strace, если что? :)

Я всё равно зааппрувил, т.к. это мелочи уже.
